README.md

```diff
@@ -27,8 +27,8 @@ We're hoping that DecisionBox enables you to demonstrate that your app is making
 ### Quick Walk-Through
 
 1.	Install the DecisionBox SDK and create your first Decision site.  You'll be asked to give this site a name, and a default 'classifier' will be created for you, called "passthrough."   This is actually a passthrough to your hosted LLM provider, so you'll let your app run and collect some data using this method, before replacing this line of code with a local classifier.  The difference is that all generated responses will recorded, so you can label them, and this will reveal your baseline accuracy to improve upon.
-2.	Under your Decision Site, create a new Classifier with the appropriate outputs (choices).  You'll give this classifier a name and make this "active" rather than the default passthrough classifier. 
```

Comment:
I kind of like "active" from before, bc I think the word default is overloaded in general. For example, the passthrough was created "by default", but the user can choose which classifier should be "active".

if we kept the word default, ppl might get confused. for example:
"what was the default classifier?", "oh, the default classifier was passthrough", "no, i mean what is default now?"

if you use the word active, ppl could talk about the classifiers more clearly

Reply:
good points!  I liked 'active' too as a concept.  But I think the direction is to render meaningless the need for any given classifier to be designated active, ie, you can use any of them at any time, and labeled data will always show what classifier was used at the time of the prediction.   So many can be 'ready' at once.

But the need remains to identify what classifier will be used if none is explicitly specified, ie 'default' -- BTW I can't think of a synonym for default but if we have a better term, let's use that. 

Imma check my  understanding and return here.

Reply:
Follow up in https://linear.app/blueteam/issue/BT-892/revamp-readme-and-docs-to-be-consistent-with-oss-user-expectations